config.go

```diff
@@ -1794,6 +1794,48 @@ func parseRPCParams(cConfig *lncfg.Chain, nodeConfig interface{},
 			}
 		}
 
+		// Get the daemon name for displaying proper errors.
+		switch net {
+		case chainreg.BitcoinChain:
+			daemonName = "bitcoind"
+			confDir = conf.Dir
+			confFile = conf.ConfigPath
+			confFileBase = "bitcoin"
+		case chainreg.LitecoinChain:
```

Comment:
I think that we can already delete litecoin-related code? @guggero

Reply:
I think it's something we should do for 0.16, since it's basically already broken, but can't hurt to just live it in for now ;)